layers/best_practices/best_practices_utils.cpp

```diff
@@ -141,7 +141,7 @@ const char* DepReasonToString(ExtDeprecationReason reason) {
     }
 }
 
-bool BestPractices::ValidateDeprecatedExtensions(const char* api_name, const char* extension_name, uint32_t version,
+bool BestPractices::ValidateDeprecatedExtensions(const char* api_name, const char* extension_name, APIVersion version,
```

Comment:
Yes, all of these have been merged upstream already in the first upstream MR, together with the generator script changes generating this class.

The only thing that broke since the last upstream is the chance to customize the implementation of APIVersion, but that was redone in the new upstream PR.